### PR TITLE
Create user screen

### LINKED ISSUE #65: Create user screen

### PR DESCRIPTION
**Pull request by icerrate**
Close #65

### CODE REVIEW

File: app/src/main/res/layout/fragment_login.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
     android:paddingLeft="@dimen/dimen_6_5"
     android:paddingRight="@dimen/dimen_6_5"
     android:scrollbars="none"
-    tools:context="com.belatrixsf.allstars.ui.login.LoginFragment">
+    tools:context="com.belatrixsf.allstars.ui.signup.SignupFragment">
```
Comment (icerrate) [inline app/src/main/res/layout/fragment_login.xml]: it must be "login.LoginFragment"

File: app/src/main/res/layout/fragment_signup.xml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/white"
+    android:fillViewport="true"
+    android:paddingLeft="@dimen/dimen_6_5"
+    android:paddingRight="@dimen/dimen_6_5"
+    android:scrollbars="none"
+    tools:context="com.belatrixsf.allstars.ui.login.LoginFragment">
```
Comment (icerrate) [inline app/src/main/res/layout/fragment_signup.xml]: it must be "signup.SignupFragment"

File: app/src/main/java/com/belatrixsf/allstars/networking/retrofit/requests/CreateEmployeeRequest.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+/**
+ * Created by icerrate on 16/05/16.
+ */
+public class CreateEmployeeRequest {
+
+    private String email;
+
+    public CreateEmployeeRequest(String username) {
+        this.email = username;
+    }
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/allstars/networking/retrofit/requests/CreateEmployeeRequest.java:32]: username?

File: app/src/main/java/com/belatrixsf/allstars/ui/login/LoginPresenter.java [hunk 4]
```diff
@@ -63,4 +63,8 @@ public void onFailure(ServiceError serviceError) {
             }
         });
     }
+
+    public void signup() {
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/allstars/ui/login/LoginPresenter.java]: signUppppp

File: app/src/main/java/com/belatrixsf/allstars/ui/login/LoginFragment.java [hunk 5]
```diff
@@ -47,6 +49,7 @@
     @Bind(R.id.username) EditText usernameEditText;
     @Bind(R.id.password) EditText passwordEditText;
     @Bind(R.id.log_in) Button logInButton;
+    @Bind(R.id.sign_up) TextView signUpTextView;
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/allstars/ui/login/LoginFragment.java:55]: this good

File: app/src/main/java/com/belatrixsf/allstars/ui/login/LoginFragment.java [hunk 6]
```diff
@@ -110,6 +119,11 @@ public void loginClicked() {
         loginPresenter.login(username, password);
     }
 
+    @OnClick(R.id.sign_up)
+    public void signupClicked() {
+        loginPresenter.signup();
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/allstars/ui/login/LoginFragment.java]: this bad

File: app/src/main/java/com/belatrixsf/allstars/ui/login/LoginView.java [hunk 7]
```diff
@@ -28,6 +28,7 @@
 public interface LoginView extends AllStarsView {
 
     void goHome();
+    void goSignup();
     void enableLogin(boolean enable);
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/allstars/ui/login/LoginView.java:31]: camelcase

File: app/src/main/java/com/belatrixsf/allstars/ui/signup/SignUpActivity.java [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+import com.belatrixsf.allstars.R;
+import com.belatrixsf.allstars.ui.common.AllStarsActivity;
+
+/**
+ * Created by icerrate on 16/05/16.
+ */
+public class SignupActivity extends AllStarsActivity {
+
+    @Override
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/allstars/ui/signup/SignUpActivity.java:33]: SignUp

File: app/src/main/java/com/belatrixsf/allstars/ui/signup/SignupPresenter.java [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+
+    @Inject
+    public SignupPresenter(SignupView view, EmployeeService employeeService) {
+        super(view);
+        this.employeeService = employeeService;
+    }
+
+    public void checkIfEmailIsValid(String email) {
+        view.enableSend(!TextUtils.isEmpty(email) && email.contains("@belatrixsf.com"));
+    }
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/allstars/ui/signup/SignupPresenter.java]: remove this placeholder belatrix email validation which is not the original to our company

File: app/src/main/res/layout/fragment_login.xml [hunk 10]
```diff
@@ -74,6 +74,18 @@
...
 
+        <TextView
+            android:id="@+id/sign_up"
+            android:layout_height="wrap_content"
+            android:layout_width="match_parent"
+            android:textSize="@dimen/text_16"
+            android:textColor="@color/colorAccent"
+            android:gravity="center"
+            android:padding="10dp"
+            android:clickable="true"
```
Comment (PedroCarrillo) [inline app/src/main/res/layout/fragment_login.xml]: 10dp

File: app/src/main/java/com/belatrixsf/allstars/ui/signup/SignupFragment.java [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+        signUpPresenter.signUp(email);
+    }
+
+    @Override
+    public void showMessage(String message){
+        DialogUtils.createInformationDialog(getActivity(), message, getString(R.string.app_name), new DialogInterface.OnClickListener() {
+            @Override
+            public void onClick(DialogInterface dialog, int which) {
+                backToLogin();
+            }
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/allstars/ui/signup/SignupFragment.java]: ?? should call presenter

**Pull request merged**